README.md

```diff
@@ -101,8 +102,23 @@ The released version of the SDK can be downloaded from https://github.com/Appdyn
     * [C++ SDK and Sample App](https://github.com/Appdynamics/iot-cpp-sdk)
     * [Python Sample Code](https://github.com/Appdynamics/iot-rest-api-sample-apps)
 
+## Miscellaneous
+* Code Coverage 
```

Comment:
Delete this line as the section should be called "Code Coverage".

Reply:
I plan to add info about jacoco [Code Coverage], findbugs[Code Quality], checkstyle[Code Style] in this section hence I named it miscellaneous. @zhouyaoji 
Any other recommendations?